TipCatDotNet.Api/Services/Payments/PaymentService.cs

```diff
@@ -98,7 +96,7 @@ private async Task<Result<PaymentIntent>> GetPaymentIntent(string paymentIntentI
             StripeConfiguration.ApiKey = _paymentSettings.StripePrivateKey;
 
             var service = new PaymentIntentService();
-            var paymentIntent = await service.GetAsync(paymentIntentId);
+            var paymentIntent = await service.GetAsync(paymentIntentId, null , null, cancellationToken);
```

Comment:
If second and third parameters are optional, you may type it like `GetAsync(paymentIntentId, cancellationToken: cancellationToken)`